File: src/app/model/unit.ts
import { Restaurant } from "./retaurant";
import { TypeUnit } from "./typeunit";
import { User } from "./user";

export interface Unit{
    id:number,
    name: string,
    description:string,
    created:Date,
    updated:Date,
    deleted:boolean,
    unitType:TypeUnit,
    restaurantDTO:Restaurant,
    createdUser:User["createdUser"],
   updatedUser:User["updatedUser"],
}

File: QuanlyNhahang_API/Model/Unit.cs
using System;
using System.ComponentModel.DataAnnotations;
namespace QuanlyNhahang_API.Model
{
    public class Unit
    {
        [Key]
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime Created { get; set; }
        public DateTime Updated { get; set; }
        public bool Deleted { get; set; }
        public virtual Restaurant Restaurant { get; set; }
        public virtual UnitType UnitType { get; set; }
        public virtual User CreatedUser { get; set; }
        public virtual User UpdatedUser { get; set; }
    }
}